generic-examples/languages/README.md

```diff
@@ -1,3 +1,23 @@
-# Supported languages Camel K examples
+# Supported languages for writing Camel K integrations
 
-Find useful examples about how to develop a Camel K integration in `Java`, `Javascript`, `Groovy`, `XML` and `YAML`.
\ No newline at end of file
+Contains useful examples on how to develop a Camel K integration in `Java`, `Javascript`, `Groovy`, `XML`, `Kotlin` and `YAML`.
+
+You can find more information about Apache Camel and Apache Camel K on the [official Camel website](https://camel.apache.org).
+
+## Before you begin
+
+Read the general instructions in the [root README.md file](/README.md) for setting up your environment and the Kubernetes cluster before looking at this example.
+
+Make sure you've read the [installation instructions](https://camel.apache.org/camel-k/latest/installation/installation.html) for your specific
+cluster before starting the example.
+
+## Understanding the examples
+- [routes.kts](./routes.kts): defines a route that logs a message every second
+- [routes.yaml](./hello.xml): defines a route that every 5 secs, transforms the message content to uppercase and logs it.
+- [simple.js](./simple.js): logs message periodically, the multiline query parameter is set to true, thus each information is output on a newline
```

Comment:
There are more examples here. Why only these three?

Reply:
I felt like the ones I left out were a bit similar to the ones I described. But you are right, we are showing the supported languages, so they should all be described. Done!